tendermint/src/vote.rs

```diff
@@ -101,7 +101,9 @@ impl lite::Vote for SignedVote {
     }
 
     fn sign_bytes(&self) -> Vec<u8> {
-        AminoMessage::bytes_vec(&self.vote.to_owned())
+        let mut buf = Vec::with_capacity(self.vote.encoded_len() + 4);
```

Comment:
This should work (given the right imports):
```suggestion
        let vote_len = self.vote.encoded_len();
        let len_delimiter_len =
            prost_amino::encoding::encoded_len_varint(vote_len.try_into().unwrap());
        let mut buf = Vec::with_capacity(self.vote.encoded_len() + len_delimiter_len);
```
I think it would be better to add a method to the `AminoMessage` trait. Sth. like `length_delimited_bytes_vec`. But OK to merge for now.